src/enum.lisp

```diff
@@ -27,6 +27,16 @@
 
 (in-package #:cffi)
 
+;; TODO the accessors names are rather inconsistent:
+;; FOREIGN-ENUM-VALUE           FOREIGN-BITFIELD-VALUE
+;; FOREIGN-ENUM-KEYWORD         FOREIGN-BITFIELD-SYMBOLS
+;; FOREIGN-ENUM-KEYWORD-LIST    FOREIGN-BITFIELD-SYMBOL-LIST
+;; I'd rename them to: FOREIGN-*-KEY(S) and FOREIGN-*-ALL-KEYS -- attila
```

Comment:
the problem with -NAME is that enums themselves have names, so it should be -ENTRY-NAME(S) then, which is kinda long.

re plural: i'd use -KEY and -KEYS as needed.